src/walk.rs

```diff
@@ -535,6 +535,13 @@ fn spawn_senders(
                 }
             }
 
+            if config.is_printing() {
+                if let Some(ls_colors) = &config.ls_colors {
+                    // Try to compute colors in parallel
```

Comment:
I'm not sure I understand this comment ("Try to…"). The code looks to me like we *do* compute the style in parallel (since it happens here, not in the receiver). Or is there a scenario where we still need to compute it in the receiver?

Reply:
I said "try to" because I thought if it returned `None`, it would retry from `print_entry_colored()`.  However that's not how `OnceCell` works, it will cache the `None`.  I'll change the comment.